src/index.css

```
@@ -1,13 +1,768 @@
+
+.comment-score {
+  font-size: 12px;
+  color: var(--clr-text-secondary);
+}
+
+.comment-body {
+  font-size: 14px;
+  color: var(--clr-text);
+  line-height: 1.6;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)